postBuild

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
I do not know if there are guidelines for writing postBuild scripts, but is is good practice in general for such scripts to add
```shell
set -e
```
at the beginning of the script so that it aborts if any command fails.